root/etc/e-smith/events/actions/nethserver-freepbx-conf

```diff
@@ -34,6 +34,17 @@ echo " *** Installing FreePBX" >> /var/log/freepbx_rpm_install.log
 
 mkdir -p /var/lib/asterisk/sounds/custom/
 
+if [[ ! -f /etc/asterisk/modules.load ]]; then
+	# write modules.load file for asterisk first start
+	echo -e '[modules]\nautoload=no' > /etc/asterisk/modules.conf
+
+	# Allow installation with asterisk > 17
+	sed -i 's/if (version_compare($matches\[1\], "11", "lt") || version_compare($matches\[1\], "17", "ge")) {/if (version_compare($matches[1], "11", "lt") || version_compare($matches[1], "19", "ge")) {/' /usr/src/freepbx/installlib/installcommand.class.php
+
+	# Allow retrieve conf with asterisk > 17
+	sed -i 's/if (version_compare(\$version, "11", "lt") || version_compare(\$version, "17", "ge")) {/if (version_compare(\$version, "11", "lt") || version_compare(\$version, "19", "ge")) {/' /usr/src/freepbx/amp_conf/bin/retrieve_conf
```

Comment:
These files are part of rpm, your are going to break package integrity.
Please create a patch to be applied by the spec